src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/PrisonOffenderManager.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.digital.delius.jpa.standard.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+
+@EqualsAndHashCode(of = "prisonOffenderManagerId")
+@ToString(exclude = {"team","staff","probationArea", "responsibleOfficer" ,"managedOffender"})
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder(toBuilder = true)
+@Entity
+@Table(name = "PRISON_OFFENDER_MANAGER")
+public class PrisonOffenderManager {
+
+    @Id
+    @Column(name = "PRISON_OFFENDER_MANAGER_ID")
+    private Long prisonOffenderManagerId;
+
+    @JoinColumn(name = "ALLOCATION_TEAM_ID")
+    @OneToOne
+    private Team team;
+
+    @JoinColumn(name = "ALLOCATION_STAFF_ID")
+    @OneToOne
+    private Staff staff;
+
+    @Column(name = "OFFENDER_ID")
+    private Long offenderId;
```

Comment:
Do you need this and `managedOffender` ?

Reply:
Yes I think so. I will have a check shortly to see where it is used.